rclcpp/src/rclcpp/graph_listener.cpp

```diff
@@ -39,17 +39,15 @@ GraphListener::GraphListener(std::shared_ptr<rclcpp::Context> parent_context)
 : parent_context_(parent_context),
   is_started_(false),
   is_shutdown_(false),
-  interrupt_guard_condition_context_(nullptr),
   shutdown_guard_condition_(nullptr)
 {
   // TODO(wjwwood): make a guard condition class in rclcpp so this can be tracked
```

Comment:
What about this comment?

Reply:
Yeah, I saw that, and I think that it would simplify this class implementation.
I didn't do that as part of this PR, as this tries to be a minimal backportable change.